models.py

```diff
@@ -1,20 +1,23 @@
 from app import db
 from flask_login import UserMixin
 
-class User(UserMixin, db.Model):
+
+class Usercredential(UserMixin, db.Model):
```

Comment:
nit: should we camel PascalCase this? ie. UserCredential
Not a big deal just a styling suggestion.